src/reference/asciidoc/ip.adoc

```diff
@@ -364,6 +364,41 @@ Initially, the first factory in the configured list is used; if a connection sub
 
 NOTE: When using the failover connection factory, the singleUse property must be consistent between the factory itself and the list of factories it is configured to use.
 
+[[affinity-cf]]
```

Comment:
Looks like you have missed this change in the `ip.adoc`, since I see one in the `whats-new`.
Will fix on merge though.